_posts/2021-11-11-java-release-notes.md

```diff
@@ -299,7 +277,7 @@ try {
 ```
 In `finally` block, we would call `close()`. The `finally` block ensures that the reader is always properly closed.
 
-#### Using try-with-resource
+#### Improvement on Autoclosable
```

Comment:
"Using try-with-resources" as a section header is a superior section header to "Improvement on Autoclosable". Think about each title and what you would expect from section with that title. The expectation is much clearer for the section header with the verb ("using").